src/components/ShowRawPing/components/PingSection.js

```diff
@@ -90,7 +91,7 @@ const PingSection = ({ pingSection, header, isNested }) => {
     <>
       {isNested ? (
         <details>
-          <summary>{renderTitle()}</summary>
+          <summary onClick={() => {recordClick('PingData.summary')}}>{renderTitle()}</summary>
```

Comment:
Hmm.. I was wrong, maybe we don't need this. Mostly because I think I want to get rid of the expand/collapse feature with the headers. I think this data is more useful if it is just all displayed at once. For now though, this is fine.

If I change this feature I will do it before we deploy, so I could just take this out. But nothing for you to worry about now!

Reply:
Okay. Leaving it as it is then.